modules/apps/frontend-data-set/frontend-data-set-builder-web/src/main/java/com/liferay/frontend/data/set/builder/web/internal/application/list/FDSBuilderPanelApp.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.data.set.builder.web.internal.application.list;
+
+import com.liferay.application.list.BasePanelApp;
+import com.liferay.application.list.PanelApp;
+import com.liferay.application.list.constants.PanelCategoryKeys;
+import com.liferay.frontend.data.set.builder.web.internal.constants.FDSBuilderPortletKeys;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.Portlet;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.PropsUtil;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marko Cikos
+ */
+@Component(
+	property = {
+		"panel.app.order:Integer=150",
+		"panel.category.key=" + PanelCategoryKeys.CONTROL_PANEL_DATA_MANAGEMENT
+	},
+	service = PanelApp.class
+)
+public class FDSBuilderPanelApp extends BasePanelApp {
```

Comment:
About `FDSBuilder` naming, this is a great moment to decide. 

### The FDS
What does this app allow to build? just `FDS`? `FDSViews`?

Initially, I was leaning towards `FDSViewBuilder` as the internal prefix for portlet, panelapps, and other constants. 

However, this app is likely in charge of many more things as it's going to provide everything we need to configure FDS, not just views: basically, everything illustrated [here](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2160590867/FDS+Configuration+constituents#Other-presentational-attributes). So, FDS looks better than FDSView

### The Builder 
For sure, app will allow to build views (or FDS configurations), but also to manage them: creation, deletion, configuration and probably management of custom views associated to each dataset view.

So, `Builder` or `Manager`?

Reply:
Naming is difficult :tm:  

> FDS or FDSViews

I see FDS as the name of _component_. The name of _app_. As Brian suggested at one point, we can call it `Mango`. FDS View is a set of customizations of the component. This is what we are building. 

With that in mind, I'd say that PR is not right, we should include the View keyword.

> Builder or Manager

In general, I like `Builder` slightly more than `Manager`, because we are highlighting that we are building something. A "manager" can manage stuff that it didn't build, for example `Licence Manager`. On the other hand, I would expect any "builder" to have at least some management features. But! `Builder` has a consistent meaning in DXP, where it is something that builds and only builds (service builder, REST builder). With this in mind, `Manager` is likely better option.

There is also a third option: none of the above. If we are set on `FDSViews` in FDS or FDSViews choice, this also opens "view" as the only differentiating keyword: `frontend-data-set-views-web`, `FDSViewsPortlet` etc. This matches nicely with `FDSViewEntry` DB entity.